### PR TITLE
13 docs

### PR DESCRIPTION
**Pull request by jacobtoppm**
Adds first draft of README and further explanatory comments on classes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+
+## Basic Usage
+
+1. To enable Google Docs import for a Page model, it should inherit from `ContentImportMixin` 
+(`wagtail_content_import.models.ContentImportMixin`). By default, content will be imported into into a StreamField called body
+(see [Changing Import Fields](#Changing-Import-Fields) for how to change this).
+
+2. You'll then need to create a Mapper, which maps the parsed document into your StreamField blocks. Create a class deriving from `wagtail_content_import.mappers.streamfield.StreamFieldMapper`:
+
+    ```
```
Comment (zerolab) [inline README.md]: worth adding "python` after the 3 opening backticks - will add nice syntax highlighting

Comment (jacobtoppm) [reply README.md]: Thanks for the tip! I've added that now.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+
+4. Open the [Credentials](https://console.developers.google.com/apis/credentials) page in the API Console.
+
+5. Select `Create credentials`, then `OAuth client ID`
+
+6. If you haven't already configured the consent screen, you will need to configure this now.
+
+    1. Under `Scopes for Google APIs`, click `Add scope`.
+
+    2. Add `../auth/documents.readonly` and `..auth/drive.readonly` scopes.
```
Comment (gasman) [inline README.md]: Missing a slash before `auth` in `..auth/drive.readonly`?

Comment (jacobtoppm) [reply README.md]: Thanks, fixed!

**Pull request merged**